common-pg/src/test/scala/com/socrata/pg/soql/SqlizerJoinTest.scala

```diff
@@ -258,7 +258,7 @@ class SqlizerJoinTest  extends SqlizerTest {
 
   test("@this alias does not spread into join more") {
     val soql = "SELECT primary_type |> SELECT @c.primary_type, @x.name FROM @this as c JOIN (SELECT name, @dog.breed, @bird.bird, @f.fish FROM @dog JOIN @bird on breed=@bird.breed JOIN @fish as f on breed=@f.breed) as x ON primary_type=@x.name"
-    val expected = """SELECT "_c"."primary_type","_x"."name" FROM (SELECT "t1".primary_type_7 as "primary_type" FROM t1) as "_c" JOIN (SELECT "name_55" as "name","_dog"."breed_56" as "breed","_bird"."bird_68" as "bird","_f"."fish_68" as "fish" FROM t12 JOIN t13 ON ("breed_56" = "_bird"."breed_66")  JOIN t14 as "_f" ON ("breed_56" = "_f"."breed_66")) as "_x" ON ("_c"."primary_type" = "_x"."name")"""
+    val expected = """SELECT "_c"."primary_type","_x"."name" FROM (SELECT "t1".primary_type_7 as "primary_type" FROM t1) as "_c" JOIN (SELECT "name_55" as "name","_dog"."breed_56" as "breed","_bird"."bird_68" as "bird","_f"."fish_68" as "fish" FROM t12 JOIN t13 as "_bird" ON ("breed_56" = "_bird"."breed_66")  JOIN t14 as "_f" ON ("breed_56" = "_f"."breed_66")) as "_x" ON ("_c"."primary_type" = "_x"."name")"""
```

Comment:
This is my favorite thing ever; it's basically testing that this bug _does_ happen.

Reply:
😆